### ISSUE TITLE
Error updating xrft to version 0.2.1 from GitHub

### ISSUE THREAD

**Issue by paigem** [xgcm/xrft#108]
Hi xrft folks,

Just wanted to raise a small issue I had a couple days ago. I wanted the updated xrft version from GitHub in order to use the 'real' option in the dft function. I'm currently running on the Pangeo cloud and ended up getting it to work but in a rather round-about fashion (and with help from @navidcy who had the same issue on a different server). My steps:

- Using `pip install git+https://github.com/xgcm/xrft.git` gave error: `ModuleNotFoundError: No module named 'versioneer'`
- Then forked and cloned the repo into my Pangeo Cloud user account
- Trying to install with `pip install --user .` gave the same error as above
- Installing with `python setup.py install --user` worked!! I now have xrft version 0.2.1 running on my cloud account.

I don't really understand why some of the above steps worked and some didn't. I'm new to xrft and relatively new to GitHub as well, so it's likely I've missed something obvious here. I also realize that this issue might no longer be relevant with a new conda/pip release of the updated version.

**Comment by navidcy** [comment]
Indeed, installing from a specific git commit fails:

```python
(base) [nc3020@gadi-login-01 site-packages]$ python -m pip install git+https://github.com/xgcm/xrft.git@aad0d2de57999ff2cc5b0559767876027a12659f
Defaulting to user installation because normal site-packages is not writeable
Collecting git+https://github.com/xgcm/xrft.git@aad0d2de57999ff2cc5b0559767876027a12659f
  Cloning https://github.com/xgcm/xrft.git (to revision aad0d2de57999ff2cc5b0559767876027a12659f) to /scratch/v45/nc3020/tmp/pip-req-build-ol6sh260
  Running command git clone -q https://github.com/xgcm/xrft.git /scratch/v45/nc3020/tmp/pip-req-build-ol6sh260
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  ERROR: Command errored out with exit status 1:
   command: /g/data3/hh5/public/apps/miniconda3/bin/python /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /scratch/v45/nc3020/tmp/tmpbewd44kp
       cwd: /scratch/v45/nc3020/tmp/pip-req-build-ol6sh260
  Complete output (16 lines):
  Traceback (most recent call last):
    File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 280, in <module>
      main()
    File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 263, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py", line 114, in get_requires_for_build_wheel
      return hook(config_settings)
    File "/scratch/v45/nc3020/tmp/pip-build-env-g1t2avac/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 150, in get_requires_for_build_wheel
      config_settings, requirements=['wheel'])
    File "/scratch/v45/nc3020/tmp/pip-build-env-g1t2avac/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 130, in _get_build_requires
      self.run_setup()
    File "/scratch/v45/nc3020/tmp/pip-build-env-g1t2avac/overlay/lib/python3.7/site-packages/setuptools/build_meta.py", line 145, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 2, in <module>
      import versioneer
  ModuleNotFoundError: No module named 'versioneer'
  ----------------------------------------
ERROR: Command errored out with exit status 1: /g/data3/hh5/public/apps/miniconda3/bin/python /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages/pip/_vendor/pep517/_in_process.py get_requires_for_build_wheel /scratch/v45/nc3020/tmp/tmpbewd44kp Check the logs for full command output.
```

However, installing from pypi is all good

```python
(base) [nc3020@gadi-login-01 site-packages]$ python -m pip install xrft
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: xrft in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (0.2.0)
Requirement already satisfied: dask in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from xrft) (2.25.0)
Requirement already satisfied: docrep in /home/552/nc3020/.local/lib/python3.7/site-packages (from xrft) (0.2.7)
Requirement already satisfied: future in /home/552/nc3020/.local/lib/python3.7/site-packages (from xrft) (0.18.2)
Requirement already satisfied: numpy in /home/552/nc3020/.local/lib/python3.7/site-packages (from xrft) (1.19.1)
Requirement already satisfied: xarray in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from xrft) (0.16.0)
Requirement already satisfied: pyyaml in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from dask->xrft) (5.3.1)
Requirement already satisfied: six in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from docrep->xrft) (1.15.0)
Requirement already satisfied: setuptools>=41.2 in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from xarray->xrft) (49.2.0.post20200712)
Requirement already satisfied: pandas>=0.25 in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from xarray->xrft) (1.1.2)
Requirement already satisfied: pytz>=2017.2 in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from pandas>=0.25->xarray->xrft) (2020.1)
Requirement already satisfied: python-dateutil>=2.7.3 in /g/data/hh5/public/apps/miniconda3/envs/analysis3-20.07/lib/python3.7/site-packages (from pandas>=0.25->xarray->xrft) (2.8.1)
(base) [nc3020@gadi-login-01 site-packages]$
```

**Comment by rabernat** [comment]
Thanks a lot for this issue report! We will look into it.

Note that we also just released v0.2.1, so just doing `pip install --upgrade xrft` should get you all the latest features.

**Comment by navidcy** [comment]
I see a similar error in the Conda release [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=207407&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=536). Hopefully #109 will deal with that.
